### PR TITLE
WINUI-3339-Content-Type-AvatarView

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Addition of Content Types Page for Avatar view UserGuide(UG).

### CODE REVIEW

**Comment by Edberg-Vasan** [comment]
rebuild

File: winui/AvatarView/content-types.md [hunk 1]
```diff
@@ -7,4 +7,305 @@ control: AvatarView
 documentation: ug
 ---
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:9]: @MadhumithanJayapathi Learn about the available content types and how to set values for them in the WinUI AvatarView control.

File: winui/AvatarView/content-types.md [hunk 2]
```diff
@@ -7,4 +7,305 @@ control: AvatarView
...
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+## Content Types
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:13]: @MadhumithanJayapathi Remove the heading ## Content Types and explain what are the content types available.

File: winui/AvatarView/content-types.md [hunk 3]
```diff
@@ -7,4 +7,305 @@ control: AvatarView
...
+The `AvatarView` control allows you to display the view in five different ways:
+
+* `Default` - Displays the `Avatar1` character image when initializing without any other source such as image, initials or group.
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:16]: @MadhumithanJayapathi The AvatarView control supports five different types of content: Default, Initials, CustomImage, AvatarCharacter, and GroupView.

File: winui/AvatarView/content-types.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md:9]: @Manivannan-E  Changed the description as mentioned.

File: winui/AvatarView/content-types.md (hunk 3 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content as mentioned.

File: winui/AvatarView/content-types.md [hunk 4]
```diff
@@ -1,10 +1,267 @@
 ---
 layout: post
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:2]: @MadhumithanJayapathi Change the image names as we discussed.

File: winui/AvatarView/content-types.md [hunk 5]
```diff
@@ -1,10 +1,267 @@
...
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+The AvatarView control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing without the initials, custom image, or group view types.
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:16]: @MadhumithanJayapathi Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.

File: winui/AvatarView/content-types.md [hunk 6]
```diff
@@ -1,10 +1,267 @@
...
+
+{% tabs %}
+{% highlight xaml %}
+
+<Grid>
+<syncfusion:SfAvatarView />
+</Grid>
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:23]: @MadhumithanJayapathi remove the unwanted code snippet and comments.

File: winui/AvatarView/content-types.md [hunk 7]
```diff
@@ -1,10 +1,267 @@
...
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+Initials content type is used to show initials to the `AvatarView`. When using the `ContentType` as initials, you need to set the initial character using the following properties:
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:39]: @MadhumithanJayapathi The initials content type is used to display initials in the `AvatarView`. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. Depending on the specified `InitialsType` property, the initials will be formatted accordingly.

File: winui/AvatarView/content-types.md [hunk 8]
```diff
@@ -1,10 +1,267 @@
...
+* `InitialsType` - Defines the type of characters to be displayed.  
+* `AvatarName` - Defines the initials to be displayed.
+
+### InitialsType
+
+The `InitialsType` contains the following two types:
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:45]: @MadhumithanJayapathi The InitialsType property includes two types:

SingleCharacter - used to display a single character in the avatar.
DoubleCharacter - used to display two characters in the avatar.

File: winui/AvatarView/content-types.md [hunk 9]
```diff
@@ -1,10 +1,267 @@
...
+
+* SingleCharacter
+* DoubleCharacter
+
+You must set the `AvatarName` string property for displaying the initials value in the AvatarView.
+
+**SingleCharacter**
+
+The `SingleCharacter` is used for displaying the first character in the string you have set in the `AvatarName` property.
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:55]: @MadhumithanJayapathi The single character initials type used to display the first character of the AvatarName property value as the avatar.

File: winui/AvatarView/content-types.md [hunk 10]
```diff
@@ -1,10 +1,267 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_initials_avatarview.png)
+
+**DoubleCharacter**
+
+The `DoubleCharacter` is used for displaying a two-characters text you have set in the `AvatarName` property. If the initials consist of a single word, it will show the first and last letters of that word. If the initials contain two or more words, it will display the first letter of the first word and the first letter of the second word.
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:82]: @MadhumithanJayapathi The double character initials type is used to display a two-character text as avatar that has been set in the AvatarName property. If the `AvatarName` consist of a single word, the first and last letters of that word will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.

File: winui/AvatarView/content-types.md [hunk 11]
```diff
@@ -1,10 +1,267 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_doubleinitials_avatarview.png)
+
+## Custom Image
+
+You can add a custom image by setting the `ImageSource` property.
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:111]: @MadhumithanJayapathi You can add any custom image as the avatar by using the `ImageSource` property.

File: winui/AvatarView/content-types.md [hunk 12]
```diff
@@ -1,10 +1,267 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom image.](avatarview_images/winui_imagesource_avatarview.png)
+
+## Avatar Character
+
+You can set the pre-defined images in avatar view by setting the `AvatarCharacter` property.
+
+{% tabs %}
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:137]: @MadhumithanJayapathi You can set the pre-defined avatar character images as avatar view by using the `AvatarCharacter` property.

File: winui/AvatarView/content-types.md [hunk 13]
```diff
@@ -1,10 +1,267 @@
...
+            
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with avatar character](avatarview_images/winui_avatarcharacter_avatarview.png)
+
+## Group View 
+
+You can add upto three images or initials in the same view using a GroupView type.
+
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:161]: @MadhumithanJayapathi The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties, which include:

File: winui/AvatarView/content-types.md [hunk 14]
```diff
@@ -1,10 +1,267 @@
...
+`InitialsColorMemberPath`: Sets the color for the initials in the group view.
+
+{% tabs %}
+{% highlight c# %}
+
+public class Employee
+{
+   private string name;
+
+   public string Name
```
Comment (Manivannan-E) [inline winui/AvatarView/content-types.md:174]: @MadhumithanJayapathi use single to declare the property like below.

 public Color Background { get; set; }

File: winui/AvatarView/content-types.md (hunk 4 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E  Changed the image names as we discussed.

File: winui/AvatarView/content-types.md (hunk 5 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the default as mentioned.

File: winui/AvatarView/content-types.md (hunk 6 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Removed the unwanted code snippet and comments.

File: winui/AvatarView/content-types.md (hunk 8 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the initials as mentioned.

File: winui/AvatarView/content-types.md (hunk 10 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the double character as mwntioned.

File: winui/AvatarView/content-types.md (hunk 11 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the custom image as mentioned.

File: winui/AvatarView/content-types.md (hunk 12 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the avatarCharacter as mentioned.

File: winui/AvatarView/content-types.md (hunk 13 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Changed the content of the group as mentioned.

File: winui/AvatarView/content-types.md (hunk 14 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/content-types.md]: @Manivannan-E Used the property as mentioned.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Avatar-view/content-types.md [hunk 15]
```diff
@@ -0,0 +1,235 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:2]: Layout: Post

File: winui/Avatar-view/content-types.md [hunk 16]
```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:3]: Title: Content Types in WinUI AvatarView control | Syncfusion

File: winui/Avatar-view/content-types.md [hunk 17]
```diff
@@ -0,0 +1,235 @@
...
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:4]: Description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.

File: winui/Avatar-view/content-types.md [hunk 18]
```diff
@@ -0,0 +1,235 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:5]: Platform: WinUI

File: winui/Avatar-view/content-types.md [hunk 19]
```diff
@@ -0,0 +1,235 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:6]: Control: AvatarView

File: winui/Avatar-view/content-types.md [hunk 20]
```diff
@@ -0,0 +1,235 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:7]: Documentation: UG

File: winui/Avatar-view/content-types.md [hunk 21]
```diff
@@ -0,0 +1,235 @@
...
+---
+
+# Content Types in WinUI AvatarView
+
+The `AvatarView` control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:12]: The `AvatarView` control supports five types of content: `Default,` `Initials,` `CustomImage,` `AvatarCharacter,` and `GroupView.`

File: winui/Avatar-view/content-types.md [hunk 22]
```diff
@@ -0,0 +1,235 @@
...
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:16]: The default content type displays the `Avatar1` character image when initializing the control without the initials, custom image, or group source.

File: winui/Avatar-view/content-types.md [hunk 23]
```diff
@@ -0,0 +1,235 @@
...
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+The Initials content type is used to display initials in the AvatarView. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. Depending on the specified `InitialsType` property, the initials will be formatted accordingly.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:35]: The Initials content type displays initials in the AvatarView. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. The initials will be formatted accordingly depending on the specified `InitialsType` property.

File: winui/Avatar-view/content-types.md [hunk 24]
```diff
@@ -0,0 +1,235 @@
...
+
+### InitialsType
+
+The `InitialsType` property includes two types:
+
+SingleCharacter - used to display a single character in the avatar.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:41]: SingleCharacter - Used to display a single character in the avatar.

File: winui/Avatar-view/content-types.md [hunk 25]
```diff
@@ -0,0 +1,235 @@
...
+DoubleCharacter - used to display two characters in the avatar.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:42]: DoubleCharacter - Used to display two characters in the avatar.

File: winui/Avatar-view/content-types.md [hunk 26]
```diff
@@ -0,0 +1,235 @@
...
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_single_character_initialstype_avatarview.png)
+
+**DoubleCharacter**
+
+The double character initials type is used to display a two-character text as avatar that has been set in the `AvatarName` property. If the AvatarName consist of a single word, the first and last letters of that word will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:73]: The double character initials type displays a two-character text as an avatar set in the `AvatarName` property. If the AvatarName consists of a single word, that word's first and last letters will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.

File: winui/Avatar-view/content-types.md [hunk 27]
```diff
@@ -0,0 +1,235 @@
...
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Custom image
+
+You can add any custom image as the avatar by using the `ImageSource` property.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:102]: Add any custom image as the avatar by using the `ImageSource` property.

File: winui/Avatar-view/content-types.md [hunk 28]
```diff
@@ -0,0 +1,235 @@
...
+avatarView.ImageSource = new BitmapImage(new Uri("ms-appx:///Images\\person.png"));
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom image](avatarview_images/winui_imagesource_avatarview.png)
+
+## Avatar character
+
+You can set the pre-defined avatar character images as avatar view by using the `AvatarCharacter` property.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:127]: Set the pre-defined avatar character images as avatar view by using the `AvatarCharacter` property.

File: winui/Avatar-view/content-types.md [hunk 29]
```diff
@@ -0,0 +1,235 @@
...
+avatarView.AvatarCharacter = AvatarCharacter.Avatar15;
+            
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with avatar character](avatarview_images/winui_avatar_character_avatarview.png)
+
+## Group view 
+
+The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties, which include:
```
Comment (Chrispineagunja) [inline winui/Avatar-view/content-types.md:152]: The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties.

File: winui/Avatar-view/getting-started.md [hunk 30]
```diff
@@ -20,7 +20,7 @@ This section explains the steps required to add the WinUI AvatarView control wit
 
 ## Initialize AvatarView 
 
-By default, AvatarView control is displayed with `Avatar1` character image, AvatarShape of `Circle and AvatarSize of `Large`.
+By default, AvatarView control is displayed with `Avatar1` character image, AvatarShape of `Circle` and AvatarSize of `Large`.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/getting-started.md:23]: By default, the AvatarView control is displayed with the `Avatar1` character image, AvatarShape of `Circle,` and AvatarSize of `Large.`

**Pull request merged**